com.krioltech.fnmap.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.krioltech.fnmap
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=xdg-documents
+rename-desktop-file: fnmap.desktop
+#rename-icon: fnmap
+#copy-icon: true
+#rename-appdata-file: fnmap.appdata.xml
+
+modules:
+  - buildsystem: autotools
+    name: nmap
+    sources:
+      - type: archive
+        url: https://nmap.org/dist/nmap-7.94.tgz
+        sha256: 7ebc3a0d369d5965ba8b6513bce3ff6d3307a7cc87dd18f70d0af5d8e66a2849
+  - buildsystem: simple
+    name: fnmap
+    build-commands:
+      - cp -R fnmap-archive/bin/bundle /app/fnmap
+      - chmod +x /app/fnmap/fnmap
+      - mkdir -p /app/bin/
+      - mkdir -p /app/lib/
+      - mkdir -p /app/share/icons
+      - ln -s /app/fnmap/fnmap /app/bin/fnmap
+      - install -Dm644 fnmap-archive/com.krioltech.fnmap.metainfo.xml /app/share/appdata/com.krioltech.fnmap.appdata.xml
+      - install -Dm644 fnmap-archive/fnmap.desktop /app/share/applications/fnmap.desktop
+      - install -Dm644 fnmap-archive/icons/fnmap.png /app/share/icons/com.krioltech.fnmap.png
+      - install -Dm644 fnmap-archive/icons/nmap.png /app/share/icons/com.krioltech.nmap.png
+    sources:
+      - type: archive
+#        path: ../../fnmap-flatpak/archives/fnmap-1.2.1.x64.tar.gz
+        url: https://github.com/gmckesey/fnmap-flatpak/raw/main/archives/fnmap-1.2.1.x64.tar.gz
```

Comment:
you should NEVER checkout from a git branch  (`main` in that case)

Reply:
_fixed_ I am now using the v1.2.1 tag.